src/components/dashboard/containers/topNav/index.js

```diff
@@ -99,7 +150,23 @@ class TopNav extends React.Component {
                             />
                         </StyledAccount>
                     </StyledTopNavLi>
-                    <StyledTopNavLi className="account">
+                    <StyledTopNavLi className={this.props.showAccInfo ? "navLinkRight" : "navLinkRight hidden"}>
+                        <StyledTopNavLinkRight
+                            title="Notifications"
+                            to=""
```

Comment:
is needed?

Reply:
- if you mean the `to` property than yes i have to specify.
- if you mean whether a `StyledTopNavLinkRight` is needed -probably not, i though it might link somewhere in the future...

Reply:
- if you mean the `to` property than yes.
- if you mean whether the `StyledTopNavLinkRight` is needed - i though it might link somewhere in the future and did not know what features we'll add to it, so i though it might be a good idea to reuse it.